link/res/drawable/stripe_link_arrow.xml

```diff
@@ -4,30 +4,30 @@
     android:viewportWidth="18"
     android:viewportHeight="12">
     <path
-        android:pathData="M12.45,0.188C12.199,-0.063 11.792,-0.063 11.541,0.188C11.29,0.44 11.29,0.847 11.541,1.098L15.857,5.571L16.286,6L15.857,6.643C15.429,7.286 11.541,10.902 11.541,10.902C11.29,11.153 11.29,11.56 11.541,11.811C11.792,12.063 12.199,12.063 12.45,11.812L17.812,6.455C17.932,6.334 18,6.171 18,6C18,5.829 17.932,5.666 17.812,5.545L12.45,0.188Z"
-        android:fillColor="#000000" />
+        android:pathData="M12.017,0.491C11.779,0.253 11.393,0.253 11.155,0.491C10.917,0.729 10.917,1.115 11.155,1.353L15.247,5.594L15.653,6L15.247,6.609C14.84,7.219 11.155,10.647 11.155,10.647C10.917,10.885 10.917,11.271 11.155,11.509C11.393,11.747 11.779,11.747 12.017,11.509L17.099,6.431C17.213,6.317 17.278,6.162 17.278,6C17.278,5.838 17.213,5.683 17.099,5.569L12.017,0.491Z"
+        android:fillColor="#000000"/>
     <path
-        android:pathData="M12,3.429L3.429,3.429"
-        android:strokeWidth="1.5"
+        android:pathData="M11.59,3.562L3.465,3.562"
+        android:strokeWidth="1.21875"
         android:fillColor="#00000000"
         android:strokeColor="#000000"
-        android:strokeLineCap="round" />
+        android:strokeLineCap="round"/>
     <path
-        android:pathData="M12,3.429V0.857"
-        android:strokeWidth="1.5"
+        android:pathData="M11.59,3.563V1.125"
+        android:strokeWidth="1.21875"
         android:fillColor="#00000000"
         android:strokeColor="#000000"
-        android:strokeLineCap="round" />
+        android:strokeLineCap="round"/>
     <path
-        android:pathData="M12,11.143V8.571"
-        android:strokeWidth="1.5"
+        android:pathData="M11.59,10.875V8.438"
+        android:strokeWidth="1.21875"
         android:fillColor="#00000000"
         android:strokeColor="#000000"
-        android:strokeLineCap="round" />
+        android:strokeLineCap="round"/>
     <path
-        android:pathData="M3.429,7.821C3.014,7.821 2.679,8.157 2.679,8.571C2.679,8.986 3.014,9.321 3.429,9.321V7.821ZM12,7.821H3.429V9.321H12V7.821Z"
-        android:fillColor="#000000" />
+        android:pathData="M3.465,7.828C3.129,7.828 2.856,8.101 2.856,8.438C2.856,8.774 3.129,9.047 3.465,9.047V7.828ZM11.59,7.828H3.465V9.047H11.59V7.828Z"
+        android:fillColor="#000000"/>
     <path
-        android:pathData="M0.857,5.25C0.443,5.25 0.107,5.586 0.107,6C0.107,6.414 0.443,6.75 0.857,6.75L0.857,5.25ZM12,5.251L0.857,5.25L0.857,6.75L12,6.751L12,5.251Z"
-        android:fillColor="#000000" />
+        android:pathData="M1.028,5.391C0.691,5.391 0.418,5.663 0.418,6C0.418,6.336 0.691,6.609 1.028,6.609L1.028,5.391ZM11.59,5.392L1.028,5.391L1.028,6.609L11.59,6.61L11.59,5.392Z"
+        android:fillColor="#000000"/>
```

Comment:
Updated the arrow icon using the representation from Figma. Reduced boldness compared to one on master.